platform/jvm/capture/src/main/kotlin/io/bitdrift/capture/events/performance/JankStatsMonitor.kt

```diff
@@ -92,6 +89,12 @@ internal class JankStatsMonitor(
         }
     }
 
+    /**
+     * Will only report frames as Jank if duration is at least the configured value at `MIN_JANK_FRAME_THRESHOLD_MS` (which is a default of 16 ms)
+     * This is given that depending on device characteristics, frame can be reported as Jank even the duration is below 16 ms
+     */
+    private fun FrameData.isJankAndMinDurationReached(): Boolean = isJank && minJankFrameThresholdMs > this.durationToMilli()
```

Comment:
so we're no longer checking `runtime.isEnabled(RuntimeFeature.DROPPED_EVENTS_MONITORING` here? also if I remember my boolean logic classes that second part of a logical AND shouldn't even be evaluated if the first conditions is false no? That would make the lazy optimization unnecesary